<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EasyCode</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>890</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionRename"/>
    <addaction name="actionRemove"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="editMenu">
    <property name="title">
     <string>Edit(&amp;E)</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionFoldAll"/>
    <addaction name="actionSellctAll"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionGoto"/>
   </widget>
   <widget class="QMenu" name="settingsMenu">
    <property name="title">
     <string>Settings(&amp;S)</string>
    </property>
    <widget class="QMenu" name="menuLangueage">
     <property name="title">
      <string>Langueage</string>
     </property>
     <addaction name="actionChinese"/>
     <addaction name="actionEnglish"/>
    </widget>
    <addaction name="actionAutoWrap"/>
    <addaction name="actionFont"/>
    <addaction name="menuLangueage"/>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>View(&amp;V)</string>
    </property>
    <addaction name="actionToolBar"/>
    <addaction name="actionStatusBar"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionUser"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionCPP"/>
    <addaction name="actionJava"/>
    <addaction name="actionCSharp"/>
    <addaction name="actionPython"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="editMenu"/>
   <addaction name="settingsMenu"/>
   <addaction name="menuLanguage"/>
   <addaction name="viewMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="actionPrint"/>
   <addaction name="actionRemove"/>
   <addaction name="separator"/>
   <addaction name="actionAutoWrap"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionSizeUp"/>
   <addaction name="actionSizeDown"/>
   <addaction name="separator"/>
   <addaction name="actionGoto"/>
   <addaction name="actionFind"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Res/Icon/new.png</normaloff>:/Res/Icon/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Res/Icon/open.png</normaloff>:/Res/Icon/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Res/Icon/save.png</normaloff>:/Res/Icon/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Res/Icon/save_as.png</normaloff>:/Res/Icon/save_as.png</iconset>
   </property>
   <property name="text">
    <string>SaveAs</string>
   </property>
   <property name="toolTip">
    <string>SaveAs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionRename">
   <property name="text">
    <string>Rename</string>
   </property>
   <property name="toolTip">
    <string>Rename</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Res/Icon/delete.png</normaloff>:/Res/Icon/delete.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Res/Icon/print.png</normaloff>:/Res/Icon/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="toolTip">
    <string>Print</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Res/Icon/undo.png</normaloff>:/Res/Icon/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Res/Icon/redo.png</normaloff>:/Res/Icon/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Res/Icon/cut.png</normaloff>:/Res/Icon/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="toolTip">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Res/Icon/copy.png</normaloff>:/Res/Icon/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Res/Icon/paste.png</normaloff>:/Res/Icon/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSellctAll">
   <property name="text">
    <string>SellctAll</string>
   </property>
   <property name="toolTip">
    <string>SellctAll</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Res/Icon/replace.png</normaloff>:/Res/Icon/replace.png</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="toolTip">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionGoto">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Res/Icon/goto.png</normaloff>:/Res/Icon/goto.png</iconset>
   </property>
   <property name="text">
    <string>Goto</string>
   </property>
   <property name="toolTip">
    <string>Goto</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionAutoWrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Res/Icon/autowrap.png</normaloff>:/Res/Icon/autowrap.png</iconset>
   </property>
   <property name="text">
    <string>AutoWrap</string>
   </property>
   <property name="toolTip">
    <string>AutoWrap</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>Font</string>
   </property>
   <property name="toolTip">
    <string>Font</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSizeUp">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Res/Icon/fontsizeup.png</normaloff>:/Res/Icon/fontsizeup.png</iconset>
   </property>
   <property name="text">
    <string>SizeUp</string>
   </property>
   <property name="toolTip">
    <string>SizeUp</string>
   </property>
  </action>
  <action name="actionSizeDown">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Res/Icon/fontsizedown.png</normaloff>:/Res/Icon/fontsizedown.png</iconset>
   </property>
   <property name="text">
    <string>SizeDown</string>
   </property>
   <property name="toolTip">
    <string>SizeDown</string>
   </property>
  </action>
  <action name="actionToolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ToolBar</string>
   </property>
   <property name="toolTip">
    <string>ToolBar</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionStatusBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>StatusBar</string>
   </property>
   <property name="toolTip">
    <string>StatusBar</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionUser">
   <property name="text">
    <string>UserManual</string>
   </property>
   <property name="toolTip">
    <string>UserManual</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionChinese">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Chinese</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionFoldAll">
   <property name="text">
    <string>FoldAll</string>
   </property>
  </action>
  <action name="actionCPP">
   <property name="text">
    <string>CPP</string>
   </property>
  </action>
  <action name="actionJava">
   <property name="text">
    <string>Java</string>
   </property>
  </action>
  <action name="actionCSharp">
   <property name="text">
    <string>CSharp</string>
   </property>
  </action>
  <action name="actionPython">
   <property name="text">
    <string>Python</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
